### PR TITLE
Add suffixes to typos from "req" to "ru"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -44310,6 +44320,7 @@ requsted->requested
 requsting->requesting
 requsts->requests
 reregisteration->reregistration
+rererence->reference
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:44323]: Perhaps `reverence` too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -44861,6 +44976,12 @@ reudction->reduction
 reudctions->reductions
 reuest->request
 reuests->requests
+reuire->require
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:44979]: Perhaps `retire`, `rewire`, `refire`, `rehire`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -44943,7 +45070,13 @@ reverseed->reversed
 reversees->reverses
 reverve->reserve
 reverved->reserved
+revew->review
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45073]: Also `renew`, `resew`, `revel`, `revex`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -45104,7 +45241,11 @@ roats->roasts, roast,
 roboustness->robustness
 robustnes->robustness
 rocess->process
+rocessed->processed
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45244]: Perhaps `recessed`?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:44979]: I can find `rewire`. I couldn't find the other typos.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:45073]: I'll add `renew` and `revel`.

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:44979]: Yet `retire` seems plausible. Any way, the user will be presented two options and this typo won't be fixed automatically. That's good enough for now.

**Pull request merged**